book/src/dev/rfcs/0012-value-pools.md

```diff
@@ -0,0 +1,498 @@
+- Feature Name: value_pools
+- Start Date: 2021-06-29
+- Design PR: [ZcashFoundation/zebra#2430](https://github.com/ZcashFoundation/zebra/pull/2430)
+- Zebra Issue: [ZcashFoundation/zebra#2152](https://github.com/ZcashFoundation/zebra/issues/2152)
+
+# Summary
+[summary]: #summary
+
+This document describes how to verify the Zcash chain and transaction value pools in Zebra.
+
+# Motivation
+[motivation]: #motivation
+
+In the Zcash protocol there are consensus rules that:
+    - prohibit negative chain value pools [ZIP-209], and
+    - restrict the creation of new money to a specific number of coins in each coinbase transaction. [Spec Section 3.4](https://zips.z.cash/protocol/protocol.pdf#transactions)
+
+These rules make sure that a fixed amount of Zcash is created by each block, even if there are vulnerabilities in some shielded pools.
+
+Checking the coins created by coinbase transactions and funding streams is out of scope for this design.
+
+[ZIP-209]: https://zips.z.cash/zip-0209
+
+# Definitions
+[definitions]: #definitions
+
+- `value balance` - The total change in value caused by a subset of the blockchain.
+- `transparent value balance` - The change in the value of the transparent pool. The sum of the outputs spent by transparent inputs in `tx_in` fields, minus the sum of newly created outputs in `tx_out` fields. In coinbase transactions, add the coins newly created by the block.
```

Comment:
Nitpick: coinbase is redundant in this definition now:
```suggestion
- `transparent value balance` - The change in the value of the transparent pool. The sum of the outputs spent by transparent inputs in `tx_in` fields, minus the sum of newly created outputs in `tx_out` fields.
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2430/commits/c529e0b52b376482e4984f3a339f63be0f142b1a